test/rollbax/client_test.exs

```diff
@@ -76,6 +76,36 @@ defmodule Rollbax.ClientTest do
     refute_receive {:api_request, _body}
   end
 
+  test "rate limiting" do
+    body = %{"message" => %{"body" => "pass"}}
+
+    log =
+      capture_log(fn ->
+        :ok =
+          Client.emit(:error, System.system_time(:second), body, %{rate_limit_seconds: "1"}, %{})
+      end)
+
+    assert log =~ "unexpected API status: 429/Too Many Requests"
+
+    assert_performed_request()
+
+    :timer.sleep(100)
+
+    log =
+      capture_log(fn ->
+        :ok = Client.emit(:error, System.system_time(:second), body, %{}, %{})
+      end)
+
+    assert log =~ "ignored report due to rate limiting"
+
+    refute_receive {:api_request, _body}
+
+    :timer.sleep(1000)
```

Comment:
```suggestion
    Process.sleep(1000)
```

Reply:
I saw we use `:timer.sleep/1` in other places like `test_helpers.exs`. Do you want me to fix them as well?

Reply:
Yes please, possibly as a separate PR :)